GOVERNANCE.md

```diff
@@ -181,7 +202,9 @@ nominee to improve their contribution profile.
 
 Collaborators might overlook someone with valuable contributions. In that case,
 the contributor may open an issue or contact a collaborator to request a
-nomination.
+nomination. Such "self-nominations" follow the same process as regular
+nomination -- that is, the contributor is not nominated until an existing
+Collaborator formally proposes it and opens the discussion.
```

Comment:
We don't have consensus over this one, I don't think it makes sense to make changes to the self-nomination, AFAIK it has never been a problem, I don't see a reason to make changes.
```suggestion
nomination.
```

Reply:
I'd support removing mention of self-nomination entirely. Anyone with sufficient involvement in Node.js (where becoming a collaborator would benefit them and/or Node.js) is going to have plenty of names (or at least GitHub handles and email addresses and Slack handles) of people to ask. There is never a need to self-nominate. The self-nomination process has been entirely superfluous the entire time I've been involved in Node.js. Simplify things and get rid of it.

Reply:
> I'd support removing mention of self-nomination entirely. Anyone with sufficient involvement in Node.js (where becoming a collaborator would benefit them and/or Node.js) is going to have plenty of names (or at least GitHub handles and email addresses and Slack handles) of people to ask. There is never a need to self-nominate. The self-nomination process has been entirely superfluous the entire time I've been involved in Node.js. Simplify things and get rid of it.

https://github.com/nodejs/node/pull/57537